### PR TITLE
Bug 1261322 - [TV][Home] Move up the main card list and the folder li…

### PR DESCRIPTION
**Pull request by luke-chang**
…st when focusing on cards inside folder

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6d1d1d4a7db5e6abe5a37247f0db41bced107da8

File: tv_apps/smart-home/js/card_util.js [hunk 1]
```diff
@@ -176,6 +176,12 @@
         this._fillCardIcon(cardButton, card);
       } else if (card instanceof Folder) {
         cardButton.setAttribute('app-type', 'folder');
+
+        var folderName = document.createElement('span');
+        folderName.classList.add('folder-name');
+        this._localizeCardName(folderName, card);
```
Comment (rexboy7) [inline tv_apps/smart-home/js/card_util.js]: Why do we need this? wouldn't it generate two name span?

File: tv_apps/smart-home/style/home.css [hunk 2]
```diff
@@ -50,6 +50,7 @@ section {
   height: 33.6rem;
   top: 32.41%;
   overflow: visible;
+  transition: top 0.4s;
```
Comment (rexboy7) [inline tv_apps/smart-home/style/home.css]: May you use `transform` with `translateY` instead? That performs better.
refer:
http://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/

Comment (luke-chang) [inline tv_apps/smart-home/style/home.css]: Good catch! Thanks.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b845105c2fe0c32d617ec9db903660d4af3390b6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9967294a9a06ae4ae8b190ecccfa5cb1a79f8b05

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a03b792725e33a101860c3439dafe5e77fe41c13

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=72ce0160b07e89e52b332e13f0902670b4924d17

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1abd756a6326f0bf7fa8f04c8045892a6aaafe65

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=46ac1245100782e8bc260f2b7d5529dcae632ea3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dab8b8fb92b7c1bbb417f7bf5be392161b770c49

**Pull request merged**